launcher/tests/local-storage-key-provider.spec.ts

```diff
@@ -19,6 +19,10 @@ const NEW_WORKBENCH_FILE = `
 some code, some code, a mask to be replaced 1234567890ABCDEFGHIJKLMNOPQRSTUV, some code
 `;
 
+const NEW_WORKBENCH_FILE_NEW_LINE = `
+some code, some code, a mask to be replaced ACaZNAIlQqHJOA2XM, some code
+`;
+
```

Comment:
It looks much better now!
Since you are trying to improve getting of a secret, could you please update the `getPartOfPublicKey` method and change
```
secret += content.charAt(i * 4);
```
on
```
secret += content.charAt(i * 2);
```
??

This will allow to take a bit longer secret. So instead of `ACaZNAIlQqHJOA2XM` in the test you can use `AACNa1ZINEAAIblpQoqFHrJPOyAt2gX1`.

Reply:
And while googling for possible keyfile format I found an example like this
```
ssh-ed25519
AAAAC3NzaC1lZDI1NTE5AAAAIObHl2pRQsoCqvF7HrrUJwPcOByIAOtS2ggoX51xMWAQ eclipse@che.com
```

I'm not sure it is a right format, but I think we can try to use it as well for future.

Reply:
For the key format, ed25519 is the one GitHub seems to recommend https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

Reply:
The test are fixed with the change i * 2